pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -252,6 +252,9 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 	doc.OpenShiftCluster.Properties.ClusterProfile.PullSecret = ""
 	doc.OpenShiftCluster.Properties.ServicePrincipalProfile.ClientSecret = ""
 
+	// We don't return enriched worker profile data on PUT/PATCH operations
+	doc.OpenShiftCluster.Properties.WorkerProfilesStatus = nil
```

Comment:
it should be okay to return enriched worker profile since it's not an input

Reply:
We decided to return the **workerProfilesStatus** field only under GET/LIST operation during design meeting.

Reply:
okay